packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -110,7 +111,7 @@ const nextServerlessLoader: loader.Loader = function() {
       if (req.url.match(/_next\\/data/)) {
         sprData = true
         req.url = req.url
-          .replace(/\\/_next\\/data\\//, '/')
+          .replace(new RegExp('\\/_next\\/data\\/${escapedBuildId}\\/'), '/')
```

Comment:
```suggestion
          .replace(new RegExp('\\\\/_next\\\\/data\\\\/${escapedBuildId}\\\\/'), '/')
```

Do we really need it to be this? (since it's within another string)

Reply:
Can we do this?
```suggestion
          .replace(new RegExp('/_next/data/${escapedBuildId}/'), '/')
```